src/datadog.ts

```diff
@@ -90,6 +103,14 @@ export class Datadog extends cdk.Construct {
       this.transport.applyEnvVars(lambdaFunctions);
     }
   }
+
+  public addForwarderToLogGroups(logGroups: logs.LogGroup[]) {
```

Comment:
Maybe we should consider changing this to `addForwarderToNonLambdaLogGroups ` or something shorter but include that its made for non lambda log groups. We have the addLambdaFunctions function that takes care of lambda based log groups automatically but if someone didn't read the docs right they could see the `addForwarderToLogGroups` function and think its meant for their lambda function log groups and go through unnecessary steps with the DD CDK